Add args ldb to tikv-ctl (#3743)
tikv/tikv: docs/tools/tikv-control.md

> - This command only supports the local mode. It prints `success!` when successfully run.
> - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether the specified `region_id` is validated or not.
> - You need to run this command for all stores where specified Regions' peers are located.

> ### Ldb Cmd

Reviewer: /Cmd/Command/s

> The ldb command line tool offers multiple data access and database admin commands. Some examples are listed below. 

Reviewer: PTAL @tikv/pingcap-i18n

For more information, please consult the help message displayed when running `tikv-ctl ldb` without any arguments 
or check the documents from RocksDB.

Example data access sequence:
To dump an existing RocksDB in HEX:
```bash
$ tikv-ctl ldb --db=/tmp/test_db dump --hex > /tmp/dbdump
```
To dump the manifest of an existing RocksDB:
```bash
$ tikv-ctl ldb manifest_dump --path=/tmp/test_db/MANIFEST-000001 --json
```

> You can specify command line `--column_family=<string>` for which column family your query will be against.

Reviewer 2: ```suggestion
You can specify the column family that your query is against using the `--column_family=<string>` command line.
```

> `--try_load_options` will try to load the options file in the DB to open the DB. 

Reviewer 2: ```suggestion
`--try_load_options` loads the database options file to open the database. It is recommended to always keep this option on when the database is running. If you open the database with default options, the LSM-tree might be messed up, which cannot be recovered automatically.
```

> It is a good idea to always try to have this option on when you operate the DB. 

Reviewer 2: Remove this line and the following line.

> - This command only supports the local mode. It prints `success!` when successfully run.
> - The argument of the `-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether the specified `region_id` is validated or not.
> - You need to run this command for all stores where specified Regions' peers are located.

### Ldb Cmd

> The ldb command line tool offers multiple data access and database admin commands. Some examples are listed below. 

Reviewer 3: data accessing?

For more information, please consult the help message displayed when running `tikv-ctl ldb` without any arguments.

Example data access sequence:
```bash
$ tikv-ctl ldb --db=/tmp/test_db --create_if_missing put a1 b1
OK 
$ tikv-ctl ldb --db=/tmp/test_db get a1
b1
$ tikv-ctl ldb --db=/tmp/test_db get a2
Failed: NotFound:
$ tikv-ctl ldb --db=/tmp/test_db scan
a1 : b1
$ tikv-ctl ldb --db=/tmp/test_db scan --hex
0x6131 : 0x6231

> $ tikv-ctl ldb --db=/tmp/test_db put --key_hex 0x6132 b2

Reviewer 3: Is it proper to guide user modify the data? Users may use this tool to manipulate the data of TiKV.

Reviewer 4: This is a good point, the tool is actually quite dangerous. I think the examples here will be more useful if we can show some usual cases, like `dump` or `dump_manifest`, instead of some other commands that we rarely use.

Reviewer 4: I think we can just keep some "dump" related commands here. If we need to use other commands, just check the help message or RocksDB documents.
